src/mococrw/key.h

```diff
@@ -43,9 +43,13 @@ class AsymmetricKey
     /**
      * Supported asymmetric key types.
      */
-    enum class KeyTypes : int { RSA = EVP_PKEY_RSA, ECC = EVP_PKEY_EC };
+    enum class KeyTypes : int {
+        RSA,
```

Comment:
To keep ABI stable we could do something like:
```
enum class KeyTypes : int {
    RSA = EVP_PKEY_RSA,
    ECC = EVP_PKEY_EC,
    ECC_ED 
};
```
But ihmo this is weird and we might have to break it in the future (or we assign e.g EVP_PKEY_ED448 to ECC_ED, which is also weird because ECC_ED represents Ed448 and Ed25519).

Reply:
I think we're changing ABI at other places anyway, so let's just bump the SOVERSION and be done with it.